lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1626,11 +1626,17 @@ func (w *WalletKit) FundPsbt(_ context.Context,
 			return nil, fmt.Errorf("unknown change output type")
 		}
 
+		maxFeeRatio := chanfunding.DefaultMaxFeeRatio
+
+		if float64(req.MaxFeeRatio) != float64(0) {
```

Comment:
Hmm I wonder if there are better possibilities rather than comparing float parameters here, I think it's not really good practice hmm

Reply:
Yeah generally it's not good comparing fp values like this
e.g there could be an effective zero value that is `1e-9`, which could be produced by fp arithmetic inaccuracies

Since `req.MaxFeeRatio` is coming directly from the user provided value in the RPC, we don't really have to worry about fp calculation leftovers

Reply:
do we check the ratio for potential negative values ?

Reply:
yes, it's done on the RPC level, can add coverage